netutils/thttpd/tdate_parse.c

```diff
@@ -93,11 +93,14 @@ static int strlong_compare(const void *v1, const void *v2)
 #ifdef HAVE_DAY_OF_WEEK /* Day of week not yet supported by NuttX */
 static int strlong_search(char *str, struct strlong *tab, int n, long *lP)
```

Comment:
maybe warning can be fixed?

Reply:
I didn't go through the code, so I don't know what `lP` means or what would be a more appropriate name.
Do you have a suggestion?

Reply:
np. we can keep as is. Or just go to lower case `lp`